sdk/machinelearningservices/Azure.ResourceManager.MachineLearning/src/autorest.md

```diff
@@ -518,6 +518,9 @@ rename-mapping:
   StatusMessageLevel: JobStatusMessageLevel
   ImageType.azureml: AzureML
   ProtectionLevel: IntellectualProtectionLevel
+  OsPatchingStatus.latestPatchTime: -|date-time
```

Comment:
Consider renaming to `LatestPatchedOn`

Reply:
on generated code its LatestPatchedOn
![image](https://github.com/user-attachments/assets/f4ace0b7-ae54-4442-aa9e-3988b131fe71)